merqube_client_lib/api_client/merqube_client.py

```diff
@@ -59,9 +68,9 @@ def __init__(self, index_name: str, is_intraday: bool = False, *args: Any, **kwa
         )
 
         # get the security id for this index
-        self._sec_id = self.session.get_collection_single(f"/security/index?names={index_name}")["id"]
+        self._sec_id = self.session.get_collection_single(f"/security/index?name={index_name}")["id"]
         self._intra_sec_id = (
-            self.session.get_collection_single(f"/security/intraday_index?names={index_name}")["id"]
+            self.session.get_collection_single(f"/security/intraday_index?name={index_name}")["id"]
```

Comment:
must be careful ?names by default does not include non-prod-stage indices. 
alternatively, can set ?type=all here, but ?name= looks up an index directly.